src/constants/formatters.js

```diff
@@ -0,0 +1,9 @@
+import * as d3Format from 'd3-format';
```

Comment:
Is the consensus to move these to `src/utils`? If so, will they still be accessible?